service/src/main/java/com/ge/predix/acs/config/GraphBeanDefinitionRegistryPostProcessor.java

```diff
@@ -24,11 +24,9 @@ public void postProcessBeanFactory(final ConfigurableListableBeanFactory beanFac
     @Override
     public void postProcessBeanDefinitionRegistry(final BeanDefinitionRegistry registry) throws BeansException {
         BeanDefinition resourceRepositoryBeanDefinition = new RootBeanDefinition(GraphResourceRepository.class);
-        registry.registerBeanDefinition("resourceRepository", resourceRepositoryBeanDefinition);
         registry.registerBeanDefinition("resourceHierarchicalRepository", resourceRepositoryBeanDefinition);
 
         BeanDefinition subjectRepositoryBeanDefinition = new RootBeanDefinition(GraphSubjectRepository.class);
-        registry.registerBeanDefinition("subjectRepository", subjectRepositoryBeanDefinition);
         registry.registerBeanDefinition("subjectHierarchicalRepository", subjectRepositoryBeanDefinition);
     }
 }
```

Comment:
@irinaepshteyn  subjectHierarchicalRepository (and resource) are THE hierarchical repositories . They are used by their respective proxy repositories.  (just not wired by name)